server/src/main/java/org/apache/iotdb/db/query/udf/datastructure/tv/SerializableBinaryTVList.java

```diff
@@ -43,11 +43,12 @@ protected static int calculateCapacity(float memoryLimitInMB, int byteArrayLengt
                     * calculateSingleBinaryTVPairMemory(byteArrayLength)));
   }
 
-  protected static int calculateSingleBinaryTVPairMemory(int byteArrayLength) {
+  protected static float calculateSingleBinaryTVPairMemory(int byteArrayLength) {
     return ReadWriteIOUtils.LONG_LEN // time
         + MIN_OBJECT_HEADER_SIZE // value: header length of Binary
         + MIN_ARRAY_HEADER_SIZE // value: header length of values in Binary
-        + byteArrayLength; // value: length of values array in Binary
+        + byteArrayLength // value: length of values array in Binary
+        + 0.125F; // extra bit(1/8 Byte): whether the Binary is null
```

Comment:
We can make the `0.125` as a public static final field :D

Reply:
This field has been moved to `ReadWriteIOUtils` as a static constant.